daml-lf/interface/src/main/scala/com/digitalasset/daml/lf/iface/reader/InterfaceReader.scala

```diff
@@ -210,16 +210,16 @@ object InterfaceReader {
     } yield Some(name -> (iface.InterfaceType.Normal(DefDataType(tyVars, Variant(cons))): T))
   }
 
-  private[reader] def enum[T >: iface.InterfaceType.Normal](
+  private[reader] def `enum`[T >: iface.InterfaceType.Normal](
       name: QualifiedName,
       tyVars: ImmArraySeq[Ast.TypeVarName],
-      enum: Ast.DataEnum,
+      `enum`: Ast.DataEnum,
```

Comment:
Please avoid using back-tick. 
```suggestion
      enumeration: Ast.DataEnum,
```

Reply:
Thanks, will apply that across the board!

Reply:
> Please avoid using back-tick.

Can you also add an explanation to your request?

Reply:
> Can you also add an explanation to your request?

Here are some of my motivation : 
- Simplicity is almost always better. Why using this advance feature when a simple renaming makes the trick. 
- It is more difficult to type.
- it makes my life more complicate when I want to use regexp to search/refactor something
- intelligent does not handle them faultless. 

I prefer reserving backticks for two following usage :    
- In pattern machine for matching constant.
- to "hide" simplicity, like @S11001001 like to do.